app/data.py

```diff
@@ -50,7 +50,7 @@ def delete(self, collection: str, query: Dict):
         """ Delete existing documents in collection matching given data """
         self.collection(collection).delete_many(query)
 
-    def count(self, collection: str, query: Optional[Dict] = None) -> int:
+    def count(self, collection: str, query: Optional[Dict]) -> int:
```

Comment:
Add the `= None` after `Optional[Dict]` otherwise the function doesn't work because `Optional[Dict]` isn't captured as an empty dictionary 👍

Reply:
True, and we don't want to set it as `{}` by default because mutable defaults cause all kinds of other problems.